src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -163,7 +172,15 @@ function WorkspaceNewRoomPage(props) {
                         shouldDelayFocus={shouldDelayFocus}
                     />
                 </View>
-                <View style={styles.mb5}>
+                <View style={styles.mb2}>
+                    <Picker
+                        inputID="writeCapability"
+                        label={translate('writeCapabilityPage.label')}
+                        items={writeCapabilityOptions}
+                        defaultValue={CONST.REPORT.WRITE_CAPABILITIES.ALL}
+                    />
```

Comment:
Let's restrict this so that the picker is only present for admins. I think it would be weird if a member could create a room they're unable to post in themselves.

Reply:
I've implemented this logic. It checks is the user admin for this policy (_workspace_). Also, I moved the write capability picker down, it is under the workspace picker. It works better, because the user selects a workspace and the next picker appears below.

1. https://github.com/Expensify/App/pull/21950/commits/9e8e423ba0a95098ef6b1abbfde8b0530e21b39e - show the write capability picker for admins
2. https://github.com/Expensify/App/pull/21950/commits/3b3f3c9d53ce1b1da5aa004c1d2498126fef0094 - move the write capability picker down

<details><summary>Example</summary>
<p>

https://github.com/Expensify/App/assets/57314631/6a95fb1a-1231-4c76-b882-b0b2848dd3ce

</p>
</details>

Reply:
Okay, this looks good from a visual perspective!

I do see an edge case where a user could select "Admins only" from the picker while 1 workspace is selected, then switch to another workspace where they are not an admin, submit the form, and end up creating a room that's admins only

Reply:
@amyevans, good catch!
So, the problem occurred because `Form` still held an input reference and value. 
I've added a condition to check that this input still exists. If not, the app removes the reference and value.
Commit - https://github.com/Expensify/App/pull/21950/commits/8a224fcf64cb1d362618cf64ebbc8a873f234260

<details><summary>Example</summary>
<p>

https://github.com/Expensify/App/assets/57314631/83f16ae2-68c2-43e0-a28d-203c69dc9158

</p>
</details>

Reply:
Great, thanks!